package/scripts/cli.py

```diff
@@ -4,12 +4,20 @@
 class CLI(Script):
   def install(self, env):
     print 'Install the CDAP CLI'
+    import params
+    # Add repository file
+    helpers.add_repo(params.files_dir + params.repo_file, params.os_repo_dir)
+    # Install any global packages
+    self.install_packages(env)
+    # Install package
+    helpers.package('cdap-cli')
+    self.configure(env)
 
   def configure(self, env):
     print 'Configure the CDAP CLI'
     import params
     env.set_params(params)
-    helpers.cdap_config('client')
+    helpers.cdap_config('cli')
```

Comment:
It's more that I wasn't sure if this was correct or a typo, since the pattern of calls didn't match other similar. But if it's correct...